### PR TITLE
Reviews

### PR DESCRIPTION
**Pull request by raphaelspies**
- ReviewList Component uses Hooks for setIsLoading, setReviews, and setProductId
- Review Component renders each review (no formatting yet)
- Changed 'var' to 'let' in router for reviews

### CODE REVIEW

File: client/src/components/ReviewList.jsx [hunk 1]
```diff
@@ -26,6 +35,18 @@ class ReviewList extends React.Component {
...
+    const params = new URLSearchParams({ sort: "newest", product_id: product_id, })
+    axios({
+      method: 'GET',
+      url: `https://app-hrsei-api.herokuapp.com/api/fec2/hr-lax/reviews?${params}`,
+      headers: { 'Authorization': credentials.TOKEN },
+    })
+    .then(res => {
+      console.log(res.data.results)
+    })
+  }
```
Comment (erikeh) [inline client/src/components/ReviewList.jsx]: .catch block here to be safe?

File: apiHelpers/atelier.js [hunk 2]
```diff
@@ -1 +1,61 @@
...
+module.exports = {
+  getProducts: function(callback) {
+    axios({
+      method: 'GET',
+      url: `${endpoint}/products`,
+      headers: { 'Authorization': credentials.TOKEN },
+    })
+    .then(res => {
+      console.log(res.data)
+      callback(null, res.data)
```
Comment (erikeh) [inline apiHelpers/atelier.js]: merge issues maybe? seeing all the old callbacks re-appeared here

File: server/routes/reviews.js [hunk 3]
```diff
@@ -5,9 +5,9 @@ const API = require('../../apiHelpers/atelier')
 reviews
   .route('/:product_id/:sort_order?')
   .get((req, res) => {
-    var product_id = req.params.product_id;
+    let product_id = req.params.product_id;
     console.log(req.params)
-    var sort_order = 'newest';
+    let sort_order = 'newest';
```
Comment (erikeh) [inline server/routes/reviews.js:10]: : )

**Review by erikeh — APPROVED** [review]
Looks good! That was a lot of code to digest though haha. 

Just made a note about perhaps adding a .catch block to handle errors in one of your axios calls, but non-urgent

**Review by Lordnickim — APPROVED** [review]

File: client/src/components/ReviewList.jsx (hunk 1 above)
Comment (raphaelspies) [reply client/src/components/ReviewList.jsx]: yikes. This code is gone now that I refactored using hooks... sorry you read through all of it. Looks like I gotta learn how to use rebase

**Pull request merged**